bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/manyToManyObjectRelation.js

```diff
@@ -190,19 +190,20 @@ pimcore.object.classes.data.manyToManyObjectRelation = Class.create(pimcore.obje
         });
         this.specificPanel.add(this.fieldSelect);
 
+        this.specificPanel.add({
+            xtype: "checkbox",
+            boxLabel: t("allow_to_create_new_object"),
+            name: "allowToCreateNewObject",
+            value: this.datax.allowToCreateNewObject
+        });
+
         if(this.context == 'class') {
             this.specificPanel.add({
                 xtype: "checkbox",
                 boxLabel: t("enable_text_selection"),
```

Comment:
maybe we should move the "enable text selection" out of the class scope as well

Reply:
Yes, this would be closer to the advancedManyToManyObjectRelation then.
https://github.com/pimcore/pimcore/blob/10.5/bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/advancedManyToManyObjectRelation.js#L171